### PR TITLE
Bumping to rails 5.

### PR DESCRIPTION
**Pull request by jpmedforth**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,161 +1,179 @@
...
 
 GEM
   remote: https://rubygems.org/
   specs:
-    abstract (1.0.0)
-    actionmailer (4.2.3)
-      actionpack (= 4.2.3)
-      actionview (= 4.2.3)
-      activejob (= 4.2.3)
+    actioncable (5.0.6)
```
Comment (dim) [inline Gemfile.lock]: why not straight to 5.1?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -30,16 +30,17 @@ class ApplicationController < ActionController::Base
...
 RSpec.configure do |config|
   config.use_transactional_fixtures = true
   config.infer_spec_type_from_file_location!
   config.render_views
 
   config.before :suite do
-    silence_stream(STDOUT) do
-      ActiveRecord::Migrator.migrate(File.expand_path('../../db/migrate', __FILE__), nil)
-    end
+    # silence_stream(STDOUT) do
```
Comment (dim) [inline spec/spec_helper.rb]: drop?

File: Gemfile.lock [hunk 3]
```diff
@@ -1,161 +1,179 @@
...
+    shoulda-matchers (3.1.2)
+      activesupport (>= 4.2.0)
+
 PATH
   remote: .
   specs:
     bsm_oa (0.3.1)
-      bsm-models
-      doorkeeper (~> 3.0.0)
+      bsm-models (~> 0.9)
```
Comment (dim) [inline Gemfile.lock]: is that a dependency we can drop altogether? I doubt we really need/use it

Comment (jpmedforth) [reply Gemfile.lock]: so it does use bsm-models and that is currently at`(>= 5.0, < 5.1)` I'lll update that one now.

File: lib/bsm_oa/authorization.rb [hunk 4]
```diff
@@ -33,6 +33,7 @@ def permissions=(vals)
     protected
 
       def normalize_permissions!
+        self.permissions = [] if permissions.nil?
```
Comment (dim) [inline lib/bsm_oa/authorization.rb]: `self.permissions ||= []`?

**Review by dim — APPROVED** [review]

**Review by dim — APPROVED** [review]

**Comment by dim** [comment]
Released new gem

**Pull request merged**